src/content/docs/zh/developers/index.mdx

```diff
@@ -71,6 +71,6 @@ Scroll通过执行链下交易以及生成交易正确执行的加密证明来
 
 ## 感谢你与我们一起构建开发
 
-我们正在努力为网络带来更多的集成和支持更多的基础设施，并为我们未来的主网版本感到兴奋。
+我们正在努力为 Scroll 主网带来更多的集成和支持更多的基础设施。
```

Comment:
can revert this change, since EN version is not ready

Reply:
done